src/resources/scss/components/_card.scss

```diff
@@ -27,4 +27,45 @@
 
 .card > .card-body + .list-group > .list-group-item:first-of-type {
     border-top: 1px solid $list-group-border-color;
-}
\ No newline at end of file
+}
+
+.card {
+    background-color: white;
+}
+
+.card .card-header {
+    position: relative;
+    padding: .75rem 1.25rem;
```

Comment:
As I stated in the other PR - Use px for padding - our Gulp workflow automatically converts them to rem and we still have px as fallback for browsers that don't support rem.